ui/src/react-admin/modules/shared/types/index.ts

```diff
@@ -1,17 +1,13 @@
 import { CSSProperties } from 'react';
+import { App } from '~modules/translations/translations.core.types';
```

Comment:
Nitpick; if the `shared` module is pulling this from the `translations` module anyway, maybe it makes more sense to move `App` to `shared` too?